bundles/InstallBundle/dump/install.sql

```diff
@@ -1,585 +1,690 @@
-SET NAMES utf8mb4;
+SET
+    NAMES utf8mb4;
 
 DROP TABLE IF EXISTS `assets`;
+
 CREATE TABLE `assets` (
-  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
-  `parentId` int(11) unsigned DEFAULT NULL,
-  `type` varchar(20) DEFAULT NULL,
-  `filename` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT '',
-  `path` varchar(765) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL, /* path in utf8 (3-byte) using the full key length of 3072 bytes */
-  `mimetype` varchar(190) DEFAULT NULL,
-  `creationDate` INT(11) UNSIGNED DEFAULT '0',
-  `modificationDate` INT(11) UNSIGNED DEFAULT '0',
-  `dataModificationDate` INT(11) UNSIGNED DEFAULT NULL,
-  `userOwner` int(11) unsigned DEFAULT NULL,
-  `userModification` int(11) unsigned DEFAULT NULL,
-  `customSettings` json null,
-  `hasMetaData` tinyint(1) NOT NULL DEFAULT '0',
-  `versionCount` INT UNSIGNED NOT NULL DEFAULT '0',
-  PRIMARY KEY (`id`),
-  UNIQUE KEY `fullpath` (`path`,`filename`),
-  KEY `parentId` (`parentId`),
-  KEY `filename` (`filename`),
-  KEY `modificationDate` (`modificationDate`),
-  KEY `versionCount` (`versionCount`)
-) AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 ROW_FORMAT=DYNAMIC;
+    `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+    `parentId` int(11) unsigned DEFAULT NULL,
+    `type` varchar(20) DEFAULT NULL,
+    `filename` varchar(255) CHARACTER
+    SET
+        utf8 COLLATE utf8_bin DEFAULT '',
+        `path` varchar(765) CHARACTER
+    SET
+        utf8 COLLATE utf8_bin DEFAULT NULL,
+        /* path in utf8 (3-byte) using the full key length of 3072 bytes */ `mimetype` varchar(190) DEFAULT NULL,
+        `creationDate` INT(11) UNSIGNED DEFAULT '0',
+        `modificationDate` INT(11) UNSIGNED DEFAULT '0',
```

Comment:
the line break and indentation doesn't look right, `creationDate` and `path` should be on the same column of `filename`, should be like or so:



```suggestion
    `filename` varchar(255) CHARACTER
        SET utf8 COLLATE utf8_bin DEFAULT '',
    `path` varchar(765) CHARACTER
        SET utf8 COLLATE utf8_bin DEFAULT NULL,
        /* path in utf8 (3-byte) using the full key length of 3072 bytes */ 
    `mimetype` varchar(190) DEFAULT NULL,
    `creationDate` INT(11) UNSIGNED DEFAULT '0',
    `modificationDate` INT(11) UNSIGNED DEFAULT '0',
```

Reply:
but overall, all the indentation are somehow broken after the script/command you ran there

<img width="1068" height="116" alt="image" src="https://github.com/user-attachments/assets/847ee599-590c-4024-b2bf-eefd6d3805c5" />

eg. `index INT(11)` should be a new line and back-indented of the column of `SET` and not behind the comment
`published` not a sub-indented line of `SET` etc..